integration-tests/validation/src/main/java/com/android/designcompose/testapp/validation/MainActivity.kt

```diff
@@ -1761,6 +1777,78 @@ fun VariableBorderTest() {
     VariableBorderTestDoc.Main()
 }
 
+@DesignDoc(id = "oetCBVw8gCAxmCNllXx7zO")
```

Comment:
@timothyfroehlich I thought these were in separate files now, or is that only in the layout branch? If so what was the reason for that?

Reply:
I can do a cherry-pick and rework it if needed.

Reply:
Yes, it's only in the layout branch. The reason is that there is and will be a big diff for lots of things in Validation, so I didn't want to try to continuously handle the conflict between the two. That's part of why we agreed to do the majority of development in the rustlayout branch when we talked a few weeks ago.